### PR TITLE
feat(2719): Scheduled job stop with notification if there's no admin

### PR DESCRIPTION
**Pull request by klu909**
## Context

Queue-service needs a way to notify user when it was not able to start a scheduled job due to no admin in the pipeline
## Objective

Add an endpoint for queue-service to notify user
## References
https://github.com/screwdriver-cd/screwdriver/issues/2719

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: plugins/jobs/notify.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+module.exports = () => ({
+    method: 'POST',
+    path: '/jobs/{id}/notify',
+    options: {
+        description: 'Get a single pipeline',
+        notes: 'Returns a pipeline record',
+        tags: ['api', 'jobs'],
+        auth: {
+            strategies: ['token'],
+            scope: ['pipeline']
```
Comment (tkyi) [inline plugins/jobs/notify.js:19]: need to update `pipeline` => `job`

File: plugins/jobs/notify.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+            const pipelineId = job.pipelineId;
+
+            if (pipelineId !== credentials.pipelineId) {
+                throw boom.forbidden('Token does not have permission for this pipeline');
+            }
+
+            const pipeline = pipelineFactory.get(pipelineId);
+
+            await request.server.events.emit('job_status', {
```
Comment (tkyi) [inline plugins/jobs/notify.js:40]: should this be wrapped in a try/catch with some error logging?

Comment (klu909) [reply plugins/jobs/notify.js:40]: i am just following the pattern from place where it emits build_status

Comment (klu909) [reply plugins/jobs/notify.js:41]: and also it's just emitting an event from server, i don't think this will be the point of failure.

**Review by tkyi — DISMISSED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**